### ISSUE TITLE
website link to github

### ISSUE THREAD

**Issue by shiffman** [CodingTrain/website#420]
The website should link here (in footer, header, about page?) so people know to file issues if things are broken/missing.

**Comment by Fraasi** [comment]
Since there is no about page, I thought header would be better for this. If css takes care of aligning, fontsize etc... could it be this simple as I've done here?
https://github.com/Fraasi/Rainbow-Code/blob/master/_jekyll/includes/2-base/navigation.html  
Just added this line:
```html
<a href="https://github.com/CodingTrain/website">Github</a>
```
I don't know about jekyll, but since it is directing to another website, this should be ok.
Thoughts?

**Comment by nilsweb** [comment]
This would add a link to GitHub that looks the same way as the other links in the header. The only thing I would add is a `target="_blank"` to open it in a new tab.

**Comment by Fraasi** [comment]
Agreed. Added target and made a pull request.

**Comment by meiamsome** [comment]
Closing as fixed by #460
